### PR TITLE
[charter] add doc policy; remove CSP:EE, SRI, Suborigins, Origin Policy

### PR DESCRIPTION
**Pull request by samuelweiler**
Hoping to resolve (most of) #595

### CODE REVIEW

**Comment by samuelweiler** [comment]
Also resolves 590

File: admin/webappsec-charter-2021.html [hunk 1]
```diff
@@ -531,6 +466,15 @@ <h3>
               <p  class="draft-status"><b>Draft state:</b> Adopted from WICG</p>
   	    <p><b>Expected publication as a Candidate Recommendation:</b> No later than Q2 2022</p>	
         </dd>
+		  
+		  
+            <dt class="spec"><a href="https://wicg.github.io/document-policy/">Document Policy</a></dt>
+
+	    <dd><p> A framework for designing configurable features as part of the web platform, and for allowing web developers to configure those features as part of their site deployment.
+	      </p>
+              <p  class="draft-status"><b>Draft state:</b> Adopted from WICG</p>
```
Comment (mikewest) [inline admin/webappsec-charter-2021.html:475]: Do we need a separate CfC to publish as an FPWD? /cc @clelland

File: admin/webappsec-charter-2021.html [hunk 2]
```diff
@@ -545,6 +489,25 @@ <h3>
...
+		  
+		  	    <dt class="spec"><a href="https://www.w3.org/TR/csp-embedded-enforcement/">Content Security
+		Policy: Embedded Enforcement</a></dt>
+
+            <dd>
+              <p>A mechanism by which a web page can embed a
+		nested browsing context if and only if it agrees to
+		enforce a particular set of restrictions upon
+		itself.</p>
+		    <p>Previously published as a Working Draft, CSP:EE will be repubished as a WG note, and work will continue in WICG.</p>
```
Comment (mikewest) [inline admin/webappsec-charter-2021.html:501]: Can we kick this off?

**Review by mikewest — APPROVED** [review]
LGTM.

**Review by dveditz — APPROVED** [review]

**Review by annevk — APPROVED** [review]

**Pull request merged**